src/data/multiset/basic.lean

```diff
@@ -586,6 +586,19 @@ by simp only [repeat_succ, singleton_eq_cons, eq_self_iff_true, repeat_zero, con
 
 theorem eq_of_mem_repeat {a b : α} {n} : b ∈ repeat a n → b = a := eq_of_mem_repeat
 
+lemma repeat_left_injective (n : ℕ) (h : n ≠ 0) : function.injective (λ a : α, repeat a n) :=
```

Comment:
The proof of `list.repeat_left_injective` is a 1-liner. Does it work here (with `multiset.eq_repeat` instead of `list.eq_repeat`)?